djangocms_versioning/admin.py

```diff
@@ -143,6 +143,9 @@ def has_change_permission(self, request, obj=None):
 class ExtendedVersionAdminMixin(VersioningAdminMixin):
     """
     Extended VersionAdminMixin for common/generic versioning admin items
+
+    WARNING: This makes use of ordered fields, if custom ordering is on anything that inherits this,
```

Comment:
Warning makes this sound like something damaging, it's more of a caveat. 

```suggestion
    CAVEAT: Ordered fields are added, if custom ordering is added to any models that inherits this Mixin
```